pangeo_forge_orchestrator/routers/github_app.py

```diff
@@ -615,7 +612,7 @@ async def run(
         recipe_run.status = "in_progress"
         # Start time was first set when recipe run was queued, which could have been ages ago,
         # so if we don't update it now, we won't capture how long the pipeline actually took.
-        recipe_run.started_at = datetime.utcnow().replace(microsecond=0)
+        recipe_run.started_at = datetime.now(timezone.utc).replace(microsecond=0)
```

Comment:
Why is this better than utcnow?

Reply:
This just adds the timezone info in the string repr. However, It turns out this new version introduces a breaking change. So, i reverted back to the original version 

```python
In [4]: datetime.utcnow().replace(microsecond=0).isoformat()
Out[4]: '2022-10-24T23:52:36'
```

```python
In [7]: datetime.now(timezone.utc).replace(microsecond=0).isoformat()
Out[7]: '2022-10-24T23:53:27+00:00'
```